modules/Calendar/Cal.js

```diff
@@ -151,11 +151,11 @@ CAL.remove_edit_dialog();var cal_loaded=true;$($.fullCalendar).ready(function(){
 if(undefined!==global_colorList[element.module_name]){valueToPush["backgroundColor"]='#'+global_colorList[element.module_name].body;valueToPush["borderColor"]='#'+global_colorList[element.module_name].border;valueToPush["textColor"]='#'+global_colorList[element.module_name].text;}
 if((element['duration_hours']%24===0)&&(element['time_start']=="12:00am"||element['time_start']=="00:00")){valueToPush['allDay']=true;}
 users_activities.push(valueToPush);});constructCalendar(user_id,users_activities);});$('#calendar'+global_current_user_id).prependTo(calendarContainer);function constructCalendar(user_id,all_events){var headerFormatDayWeek='dddd D';var headerFormatMonth='dddd';var headerFormat=headerFormatDayWeek;if(global_view=='sharedMonth'||global_view=='month'){headerFormat=headerFormatMonth;}
-$('#calendar'+user_id).fullCalendar({header:{left:'',center:'',right:''},lang:global_langPrefix,views:views,minTime:global_start_time,maxTime:global_end_time,selectHelper:true,selectable:true,selectOverlap:true,slotMinutes:global_timeslots,defaultDate:global_year+"-"+global_month+"-"+global_day,editable:global_edit,disableDragging:global_items_draggable,eventLimit:true,defaultView:global_view,firstDay:global_start_week_day,height:global_basic_min_height,columnFormat:headerFormat,select:function(date,jsEvent,view){if(global_edit==true){var date_start=date.format(global_datetime_format);var date_end=jsEvent.format(global_datetime_format);var date_duration=jsEvent.diff(date);if(date.hasTime()==false){var date_end=date.add(1,'days').format(global_datetime_format);}
+$('#calendar'+user_id).fullCalendar({header:{left:'',center:'',right:''},locale:global_langPrefix,views:views,minTime:global_start_time,maxTime:global_end_time,selectHelper:true,selectable:true,selectOverlap:true,slotMinutes:global_timeslots,defaultDate:global_year+"-"+global_month+"-"+global_day,editable:global_edit,disableDragging:global_items_draggable,eventLimit:true,defaultView:global_view,firstDay:global_start_week_day,height:global_basic_min_height,columnFormat:headerFormat,select:function(date,jsEvent,view){if(global_edit==true){var date_start=date.format(global_datetime_format);var date_end=jsEvent.format(global_datetime_format);var date_duration=jsEvent.diff(date);if(date.hasTime()==false){var date_end=date.add(1,'days').format(global_datetime_format);}
```

Comment:
Please also update jssource/src_files/modules/Calendar/Cal.js and include in your pull request. I think it is around line 913